tests/run.bats

```diff
@@ -19,6 +19,7 @@ load helpers
 	buildah run        $cid cp /tmp/randomfile /tmp/other-randomfile
 	test -s $root/tmp/other-randomfile
 	cmp ${TESTDIR}/randomfile $root/tmp/other-randomfile
+	buildah run $cid -- rm -f /tmp/randomfile /tmp/other-randomfile
```

Comment:
For a start.  Also might as well add
* run buildah run $cid echo -- -n foobar
* run buildah run $cid -- echo -n foobar --
* run buildah run $cid -- echo -- -n foobar --

Just to make sure that we're not interfering with or running afoul of any reordering that the cli package is trying to do on our behalf.